mizar/common/rpc.py

```diff
@@ -212,6 +212,8 @@ def update_agent_metadata(self, ep):
         cmd = f'''{self.trn_cli_update_agent_metadata} -i \'{itf}\' -j \'{jsonconf}\''''
         logger.info("update_agent_metadata: {}".format(cmd))
         returncode, text = run_cmd(cmd)
+        if "error" in text:
```

Comment:
Is this the fix to catch the race condition where update hits before agent load?

Reply:
Yeah, the return code always returns none, but the output text will have "error" when something goes wrong.
I can turn this into a constant.